features/256-auth-gear-interaction-overview-in-next/api.md

```diff
@@ -0,0 +1,126 @@
+# auth gear interaction overview in next
+
+Base on the new product architecture decision (auth gear + cloud function and drop record gear), skygear next should bring up a new experience on how auth gear is used in cloud function and client SDK.
+
+## Goal
+
+* Keep auth gear simple, focus on auth related functionalities.
+* Keep client SDK simple, less interaction with auth gear directly.
+* More cloud functions, a developer should create corresponding cloud funtion to fulfill application logic.
+
+## Major changes
+
+* Remove admin API in client SDK.
```

Comment:
More clear should be: Move admin related features from Client SDK to APIs at Cloud Functions?